modules/reference/pages/cluster-properties.adoc

```diff
@@ -1,1796 +1,5372 @@
+
+*Default:* `null`
+
+---
+
+=== kafka_connections_max_overrides
+
+A list of IP addresses for which Kafka client connection limits are overridden and don't apply. For example, `(['127.0.0.1:90', '50.20.1.1:40']).`
+
+
+
+*Related topics*:
+
+* xref:manage:cluster-maintenance/configure-availability.adoc#limit-client-connections[Limit client connections].
+
+*Requires Restart:* No
+
+*Nullable:* No
+
+*Visibility:* `user`
+
+*Type:* array
+
+*Default:* `null`
+
+---
+
+=== kafka_connections_max_per_ip
+
+Maximum number of Kafka client connections per IP address, per broker. If `null`, the property is disabled.
+
+*Units*: number of Kafka client connections per IP address, per broker
+
+
+
+*Related topics*:
+
+* xref:manage:cluster-maintenance/configure-availability.adoc#limit-client-connections[Limit client connections].
+
+*Requires Restart:* No
+
+*Nullable:* Yes
+
+*Visibility:* `user`
+
+*Type:* integer
+
+*Accepted values:* [`0`, `4294967295`]
+
+*Default:* `null`
+
+---
+
+=== kafka_enable_authorization
+
+Flag to require authorization for Kafka connections. If `null`, the property is disabled, and authorization is instead enabled by <<enable_sasl,enable_sasl>>.
+
+Valid values:
+
+* `null`: Ignored. Authorization is enabled with <<enable_sasl,`enable_sasl`>>: `true`
+* `true`: authorization is required.
+* `false`: authorization is disabled.
+
+*Type*: boolean
+
+
+*Related properties*:
+
+* <<enable_sasl,enable_sasl>>
+* `kafka_api[].authentication_method`.
+
+*Requires Restart:* No
+
+*Nullable:* Yes
+
+*Visibility:* `user`
+
+*Type:* boolean
+
+*Default:* `null`
+
+---
+
+=== kafka_enable_describe_log_dirs_remote_storage
+
+Whether to include tiered storage as a special remote:// directory in DescribeLogDirs Kafka API requests.
+
+*Requires Restart:* No
+
+*Nullable:* No
+
+*Visibility:* `user`
+
+*Type:* boolean
+
+*Default:* `true`
+
+---
+
+=== kafka_enable_partition_reassignment
+
+Enable the Kafka partition reassignment API.
+
+*Type*: boolean
+
+
+
+*Supported versions*: Redpanda v23.1 or later.
+
+*Requires Restart:* No
+
+*Nullable:* No
+
+*Visibility:* `user`
+
+*Type:* boolean
+
+*Default:* `true`
+
+---
+
+=== kafka_group_recovery_timeout_ms
+
+Kafka group recovery timeout.
+
+*Units*: milliseconds.
+
+*Requires Restart:* No
+
+*Nullable:* No
+
+*Visibility:* `user`
+
+*Type:* integer
+
+*Accepted values:* [`-17592186044416`, `17592186044415`]
+
+*Default:* `30000ms`
+
+---
+
+=== kafka_max_bytes_per_fetch
+
+Limit fetch responses to this many bytes, even if total of partition bytes limits is higher.
+
+*Requires Restart:* No
+
+*Nullable:* No
+
+*Visibility:* `tunable`
+
+*Type:* integer
+
+*Default:* `67108864`
+
+---
+
+=== kafka_memory_batch_size_estimate_for_fetch
+
+The size of the batch used to estimate memory consumption for Fetch requests, in bytes. Smaller sizes allow more concurrent fetch requests per shard, larger sizes prevent running out of memory because of too many concurrent fetch requests.
```

Comment:
```suggestion
The size of the batch used to estimate memory consumption for fetch requests, in bytes. Smaller sizes allow more concurrent fetch requests per shard. Larger sizes prevent running out of memory because of too many concurrent fetch requests.
```